asv_bench/benchmarks/period.py

```diff
@@ -101,7 +28,7 @@ def time_second(self):
         self.per.second
 
     def time_is_leap_year(self):
-        self.per.is_leap_year
+        self.per.is_leapyear
```

Comment:
I think `is_leap_year` was actually correct ?

Reply:
Yep, you're right.  One of the two existing benchmarks used each and I guessed wrong.